react-native/react-native_2.md

```diff
@@ -0,0 +1,31 @@
+# About React and React Native components
+
+React Native requires you to have an index.[os].js for each platform. But any other component can be written once for both OSes.
+
+Components are written *the React way*:
+
+  * A component is a Javascript class (as introduced in ES6, but you can also write your code in a more traditional ES5 style).
```

Comment:
React components can be written as pure functions, classes or using React.createClass, so instead of focusing what a component is, syntaxwise, I would focus on its utility. Something in the lines of:

````
A react component is a piece of code that creates a DOM element. Unlike regular DOM elements, like <a> or <div>, a component can mix both logic and presentation. This creates a piece of code, that you can reuse through out your app, or even between projects.
````